lib/initInstance.js

```diff
@@ -42,43 +44,43 @@ const versionLocation = path.join(destination, 'component.json');
 config.productDir = path.join(__dirname, '..', 'defaults');
 
 //Begin generate any missing folders
-fs.mkdirSync(destination, {mode: initUtils.FOLDER_MODE, recursive: true});
+initUtils.mkdirp(destination, initUtils.FOLDER_MODE);
 
 if (!config.siteDir) {
   config.siteDir = path.join(destination, 'site');
 }
 const sitePluginStorage = path.join(config.siteDir, 'ZLUX', 'pluginStorage');
-fs.mkdirSync(sitePluginStorage, {mode: initUtils.FOLDER_MODE, recursive: true});
+initUtils.mkdirp(sitePluginStorage, initUtils.FOLDER_MODE);
 
 if (!config.instanceDir) {
   config.instanceDir = destination;
 }
 const instancePluginStorage = path.join(config.instanceDir, 'ZLUX', 'pluginStorage');
-fs.mkdirSync(instancePluginStorage, {mode: initUtils.FOLDER_MODE, recursive: true});
+initUtils.mkdirp(instancePluginStorage, initUtils.FOLDER_MODE);
 const recognizersPluginStorage = path.join(config.instanceDir, 'ZLUX/pluginStorage', 'org.zowe.zlux.ng2desktop/recognizers');
-fs.mkdirSync(recognizersPluginStorage, {mode: initUtils.FOLDER_MODE, recursive: true});
+initUtils.mkdirp(recognizersPluginStorage, initUtils.FOLDER_MODE);
 const actionsPluginStorage = path.join(config.instanceDir, 'ZLUX/pluginStorage/org.zowe.zlux.ng2desktop', 'actions');
-fs.mkdirSync(actionsPluginStorage, {mode: initUtils.FOLDER_MODE, recursive: true});
+initUtils.mkdirp(actionsPluginStorage, initUtils.FOLDER_MODE);
 
 const instanceConfig = path.join(config.instanceDir, 'serverConfig');
 //750 specifically, to keep server config secure
-fs.mkdirSync(instanceConfig, {mode: 0o0750});
+initUtils.mkdirp(instanceConfig, 0o0750);
```

Comment:
This was not recursive, but made recursive. 
is OK?

Reply:
this prob a bugfix

Reply:
different but harmlessly so, thanks for  spotting.